test/cases/nesting_test.rb

```diff
@@ -160,7 +160,7 @@ def test_silent_end_with_stuff
 b
 a
 HTML
-- str = "abcde"
+- str = +"abcde"
```

Comment:
Wow, I had no idea about this feature... it took me some serious digging to learn what it even was. Great work!